### PR TITLE
character modifications

### PR DESCRIPTION
**Pull request by delaynieym**
made this on my phone lol

### CODE REVIEW

File: src/commands/dia.rs [hunk 1]
```diff
@@ -7,7 +7,7 @@ fn dia(ctx: &mut Context, msg: &Message) -> CommandResult {
     if msg.author.to_string() == "<@207686242874294272>" {
         return Ok(());
     };
-    let _ = msg.channel_id.say(&ctx.http, "Did I Ask?");
+    let _ = msg.channel_id.say(&ctx.http, "D¡d | ∆sk‽");
```
[the remark this answers is not in the document]
Comment (delaynieym) [reply src/commands/dia.rs]: shouldn't be there anymore. it's from a previous commit

File: src/commands/dia.rs [hunk 2]
```diff
@@ -1,13 +1,86 @@
 use serenity::framework::standard::{macros::command, CommandResult};
 use serenity::model::prelude::*;
 use serenity::prelude::*;
+use rand::Rng;
 
 #[command]
 fn dia(ctx: &mut Context, msg: &Message) -> CommandResult {
-    if msg.author.to_string() == "<@207686242874294272>" {
+    if msg.author.to_string() != "<@483297157580062732>" {
```
[the remark this answers is not in the document]
Comment (delaynieym) [reply src/commands/dia.rs:8]: I don't see you requesting any changes for this bit...

[the remark this answers is not in the document]
Comment (captainrex493) [reply src/commands/dia.rs:8]: Change it back?

[the remark this answers is not in the document]
Comment (delaynieym) [reply src/commands/dia.rs:8]: okie boss

**Pull request merged**